src/render/direct3d12/SDL_render_d3d12.c

```diff
@@ -37,6 +37,7 @@
 #include "SDL_syswm.h"
 #include "../SDL_sysrender.h"
 #include "../SDL_d3dmath.h"
+#include <sdkddkver.h>
```

Comment:
in https://github.com/libsdl-org/SDL/commit/89a4d9ae67e82913af71a93611f396530a05f61d you avoided including this for xbox builds, but not here?

Reply:
Good point. I'll move it lower.